deploy-manage/deploy/elastic-cloud/project-settings.md

```diff
@@ -6,25 +6,81 @@ mapped_urls:
 
 # Project settings
 
-% What needs to be done: Refine
+$$$elasticsearch-manage-project-search-power-settings$$$
+
+{{serverless-full}} projects are fully managed and automatically scaled by Elastic. You have the option of {{es-serverless}}, {{observability}}, or {{elastic-sec}} for your project.
+
+Your project’s performance and general data retention are controlled by the **Search AI Lake settings**. To manage these settings:
```

Comment:
can you hint that there are add-ons available for security that are also on this page ?